accepted/2021/utf8/README.md

```diff
@@ -0,0 +1,30 @@
+# Improve UTF-8 support
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
```

Comment:
We just removed the PM/Dev markers. Please change this, rebase on top of main, and run `./update-index`  in the root.

```suggestion
**Owner** [Immo Landwerth](https://github.com/terrajobst)
```